SO_properties.py

```diff
@@ -565,16 +564,43 @@ def spin_parameter(self) -> unyt.unyt_quantity:
     def TotalInertiaTensor(self) -> unyt.unyt_array:
         """
         Inertia tensor of the total mass distribution.
+        Computed iteratively using an ellipsoid with volume equal to that of
+        a sphere with radius SORadius. Only considers particles within the SORadius
```

Comment:
Is the volume of the ellipsoid well-within the equivalent SO sphere? In other words, would we pick up particles on the edges of a heavily stretched ellipse that are beyond the SO radius?

Reply:
No we won't pick up particles outside the SO radius, and the ellipsoid is basically guaranteed to stretch outside since the initial radius is equal to the SO radius. The same happens for the projected apertures.

This is something I had considered, but there's no nice solution without more major changes to SOAP.

Reply:
Leaving for future reference: In the meeting we agreed to explore the possibility of computing shapes for apertures when we have loaded particles at larger apertures (e.g. R200 shapes when we have particles within R100).

Reply:
https://github.com/SWIFTSIM/SOAP/pull/82/commits/c0c1d4ce339c6f083e13c9d8b35b094648e26ae6